cmd/internal/server/core.go

```diff
@@ -52,8 +51,8 @@ var levelMap = map[string]log.Level{
 }
 
 // New creates a new Server.
-func New(mountpoint string, socket string, opts Opts) *Server {
-	return &Server{mountpoint: mountpoint, socket: socket, opts: opts}
+func New(mountpoint string, socket string, plugins map[string]plugin.Root, opts Opts) *Server {
```

Comment:
Is this meant to facilitate configuring which internal plugins Wash should load?

Reply:
It could be used that way. Primarily I was moving the definition of what plugins are available elsewhere, and didn't want others to rely on it as a global.

Reply:
Ah ok. Was going to suggest an `InternalPlugins`/`CorePlugins` option to keep it symmetric with `ExternalPlugins`. Bit weird to see something generic like `plugins` in the params and then have an `ExternalPlugins` option.

Reply:
We may want to move towards that symmetry when implementing #288.

Reply:
Sounds good.